src/planscape/planning/views.py

```diff
@@ -65,7 +65,7 @@ def _serialize_planning_area(planning_area: PlanningArea, add_geometry: bool) ->
 
 
 #### PLAN(NING AREA) Handlers ####
-
+@csrf_exempt
```

Comment:
are we turning off CSRF protection just for now (to get things out the door)?

Reply:
Same concern here, do we have a story to track how to get this back on afterwards? And should we add some comments on why its being turned off?

Reply:
I had turned it off just to get things going, but after looking into how to implement the change in Angular I realized it was small enough to add to this pr.

I've removed these decorators and added in a module that takes care of sending the correct tokens